server/routes/users.ts

```diff
@@ -25,5 +26,12 @@ router.put(
 	verifyUser,
 	updateUserById
 );
+router.get(
```

Comment:
same as above. It's a big security risk. We can have this endpoint. But the information returned should be limited. like we shouldn't include userId, userFirebaseAuthId, or any sensitive information.